scylla-server/src/services/runs.services.ts

```diff
@@ -0,0 +1,65 @@
+import prisma from '../prisma/prisma-client';
+
+/**
+ * CRUD operation to get all runs
+ * @returns Promise<string>  all the runs
+ */
+export const getAllRuns = async () => {
+  const data = await prisma.run.findMany();
+  return JSON.stringify(data);
+};
+
+/**
+ * CRUD operation to get run by id
+ * @param id id of run
+ * @returns Promise<json string of run>
+ */
+export const getRunById = async (id: number) => {
+  const data = await prisma.run.findUnique({
+    where: {
+      id
+    }
+  });
+
+  return JSON.stringify(data);
+};
+
+/**
+ * @param id id of run
+ * @param locationName  locationName of run
+ * @returns Promise<void>
+ */
+export const upsertRun = async (id: number, locationName: string) => {
```

Comment:
we shouldnt be upserting runs, we should just be creating them. There shouldnt be an Id